docs/src/modules/javascript/pages/views.adoc

```diff
@@ -4,18 +4,28 @@ include::ROOT:partial$include.adoc[]
 include::partial$views.adoc[]
 
 [#value-entity]
-== View from Value Entity
+== View from a Value Entity
 
-Using an example of a customer registry, you can define a `Customer` in Protobuf as:
+Consider an example of a Customer Registry service with a `customer` Value Entity. When `customer` state changes, the entire state is emitted as a value change. Value changes update any associated Views. To create a View that shows customers by their name:
```

Comment:
suggestion: "that lists customers by name"

Reply:
Done, thanks.

Reply:
Update not pushed?